R/unbump-version.R

```diff
@@ -19,7 +19,8 @@ unbump_version_impl <- function() {
 
   parent_commit_id <- gert::git_commit_info(last_commit$commit)$parent
```

Comment:
Can we extract this to a dedicated function that uses `in_example()` instead of a special environment variable?